web/craco.config.js

```diff
@@ -1,6 +1,48 @@
 const CracoLessPlugin = require("craco-less");
 
 module.exports = {
+  webpack: {

```

Comment:
Does this modify any existing work flow?

Do people need to enable gzip manually?

Reply:
This webpack configuration will be merged with the default webpack by using [webpack-merge](https://www.npmjs.com/package/webpack-merge). So we just modify the optimization phase, the rest will use the default of `craco`.

Reply:
About gzip, we need support from browsers to decompress, but with the size of our chunks, I think we are still fine.

But if we still want it, we must add this plugin [compression-webpack-plugin](https://webpack.js.org/plugins/compression-webpack-plugin/) into webpack config.